tests/main/cloud-init/task.yaml

```diff
@@ -6,8 +6,7 @@ details: |
     properly.
 
 # GCE backend sets instance data
-# TODO: extend this to openstack once we have the dev environments ready
-backends: [google]
+backends: [google, openstack]
```

Comment:
I don't recommend to add openstack in teh backends list becuase this will break the spread when it is used from master. I recommand checking is [ SPREAD_BACKEND ==  openstack ] instead.
You can skip the test incase it is not openstack or google using the "tests.exec skip-test" command